net/adguardhome/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE.txt for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=adguardhome
+PKG_VERSION:=0.102.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_SOURCE_URL:=https://github.com/AdguardTeam/AdGuardHome
+PKG_MIRROR_HASH:=c457088177907a032d67c67ce2136bbc8e462be4cf61cd4274424b5e88dfa499
+
+PKG_LICENSE:=GPL-3.0-only
+PKG_LICENSE_FILES:=LICENSE.txt
+PKG_MAINTAINER:=Dobroslaw Kijowski <dobo90@gmail.com>
+
+PKG_BUILD_DEPENDS:=golang/host node/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/AdguardTeam/AdGuardHome
+GO_PKG_EXCLUDES:=dhcpd/standalone
+GO_PKG_LDFLAGS:=-s -w
+GO_PKG_LDFLAGS_X:=main.version=$(PKG_VERSION) main.channel=release
```

Comment:
The upstream Makefile [also sets](https://github.com/AdguardTeam/AdGuardHome/blob/bab7b2dc65004d7c9661a590965f710584f063b9/Makefile#L29) `main.goarm` - do you know if this is necessary here?

Reply:
GOARM isn't set anywhere in Makefile so I've skipped it.

Reply:
For release builds, `GOARM` is set by [goreleaser](https://github.com/AdguardTeam/AdGuardHome/blob/39f2d5c4ae0f04b3598e7f96720d8872f90f1ccf/.goreleaser.yml#L32-L35).

When using make, you are expected to set the various `GO*` variables on the command line when [cross building](https://github.com/AdguardTeam/AdGuardHome#building), including `GOARM`.

Reply:
You're perfectly right! I've missed that. It should be fine now.